package.json

```diff
@@ -53,6 +53,7 @@
     "@ensdomains/resolver": "^0.2.4",
     "@ew-did-registry/claims": "0.7.1-alpha.795.0",
     "@ew-did-registry/credentials-interface": "0.7.1-alpha.795.0",
+    "@ew-did-registry/did": "0.7.1-alpha.795.0",
```

Comment:
Since this direct dependency it should be saved in package.json. Without it can not link `passport-did-auth` locally, because `npm link` installs dependency of `passport-did-auth` in `passport-did-auth` directory and thus makes `@ew-did-registry/did` unavailable for `ssi-hub`